Dockerfile.java8

```diff
@@ -6,10 +6,10 @@ COPY halconfig/packer              /opt/rosco/config/packer
 
 WORKDIR /packer
 
-RUN apk --no-cache add --update bash wget curl openssl openjdk8-jre && \
-  wget https://releases.hashicorp.com/packer/1.4.4/packer_1.4.4_linux_amd64.zip && \
-  unzip packer_1.4.4_linux_amd64.zip && \
-  rm packer_1.4.4_linux_amd64.zip
+RUN apk --no-cache add --update bash wget curl openssl openjdk8-jre git openssh-client && \
```

Comment:
Just curious, why the addition of `git` and `openssh-client`?

Reply:
@drewhodson i'm guessing this is due to a merge from upstream spinnaker i'll remove this if you desire

Reply:
I'm fine with keeping it, just curious 🙂